README.md

```diff
@@ -1,264 +1,40 @@
-* Added: Attribution-Domains support
-* Improved: Inbox validation
-* Improved: WordPress-Post-Type - Detection
-* Improved: Only validate POST params and do not fall back to GET params
-* Improved: ID handling for a better compatibility with caching plugins
-* Fixed: The "Shared Inbox" endpoint
-* Fixed: Ensure that sticky_posts is an array
-* Fixed: URLs and Hashtags in profiles were not converted
-* Fixed: A lot of small improvements and fixes
-
-See full Changelog on [GitHub](https://github.com/Automattic/wordpress-activitypub/blob/trunk/CHANGELOG.md).
-
-## Upgrade Notice ##
+> [WordPress Playground](https://wordpress.org/playground/) is the platform that lets you run WordPress instantly on any device without a host. It’s your place to build, experiment, test, and grow.
 
-### 1.0.0 ###
+## Documentation
 
-For version 1.0.0 we have completely rebuilt the followers lists. There is a migration from the old format to the new, but it may take some time until the migration is complete. No data will be lost in the process, please give the migration some time.
+WIP.
 
-## Installation ##
+## Federation
 
-Follow the normal instructions for [installing WordPress plugins](https://wordpress.org/support/article/managing-plugins/).
+ActivityPub is a protocol for federated social networks, enabling communication between different platforms. For details on what the plugin supports, refer to the [FEDERATION.md](.FEDERATION.md) file.
 
-### Automatic Plugin Installation ###
+## Contribute
 
-To add a WordPress Plugin using the [built-in plugin installer](https://codex.wordpress.org/Administration_Screens#Add_New_Plugins):
+Thank you for thinking about contributing to Jetpack! If you're unsure of anything, feel free to submit an issue or pull request on any topic. The worst that can happen is that you'll be politely directed to the best location to ask your question or to change something in your pull request. There are a variety of options for how you can help:
 
-1. Go to [Plugins](https://codex.wordpress.org/Administration_Screens#Plugins) > [Add New](https://codex.wordpress.org/Plugins_Add_New_Screen).
-1. Type "`activitypub`" into the **Search Plugins** box.
-1. Find the WordPress Plugin you wish to install.
-    1. Click **Details** for more information about the Plugin and instructions you may wish to print or save to help setup the Plugin.
-    1. Click **Install Now** to install the WordPress Plugin.
-1. The resulting installation screen will list the installation as successful or note any problems during the install.
-1. If successful, click **Activate Plugin** to activate it, or **Return to Plugin Installer** for further actions.
+* Write and submit patches.
+* If you found a bug, [file a report here](https://github.com/Automattic/wordpress-activitypub/issues/new?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen&template=bug_report.yml).
+* [Translate ActivityPub in your language](https://translate.wordpress.org/projects/wp-plugins/activitypub/).
```

Comment:
Since you use "Discussions" in this repo, maybe that's something you'd want to promote here?
https://github.com/Automattic/wordpress-activitypub/discussions

Or if you would rather have people opening issues, I guess this doesn't have to be mentioned. :)